Integrations/integration-Zendesk.yml

```diff
@@ -0,0 +1,462 @@
+    };
+
+
+    var fetchIncidents = function() {
+        var now = getLastRun();
+        var sertime;
+        var incidents = [];
+        var labels = [];
+
+        if (!now.time) {
+            sertime = getNowTime(-10);
+        } else {
+            sertime = now.time;
+        }
+
+        var att = {query: 'created>' + sertime};
+        res = sendRequest('GET', 'search.json', '', att);
+
+        if (res.results.length > 0) {
+            for (var i = 0; i < res.results.length; i++) {
+                incidents.push({
+                    name: res.results[i].subject,
+                    details: res.results[i]['description'],
+                    labels: labels,
+                    rawJSON: JSON.stringify(res.results[i]),
+                    severity: parseInt(res.results[i].priority)
+                });
+            }
+
+            setLastRun({'time': res.results[0].created_at});
+        }
+
+        return JSON.stringify(incidents);
+    }
+
+
+    switch (command) {
+        // This is the call made when pressing the integration test button.
+        case 'test-module':
+            var res = sendRequest('GET', 'users/me.json');
+
+            if (res.user.name !== 'Anonymous user' &&
+                res.user.id) {
+                return 'ok';
+            }
+
+            return JSON.stringify(res);
+
+        case 'zendesk-create-ticket':
+            return createTicket();
+
+        case 'zendesk-list-tickets':
+            return listTickets();
+
+        case 'zendesk-update-ticket':
+            return updateTicket();
+
+        case 'zendesk-ticket-details':
+            return getTicketDetails();
+
+        case 'fetch-incidents':
+            return fetchIncidents();
+        default:
+    }
+  type: javascript
+  commands:
+  - name: zendesk-create-ticket
+    arguments:
+    - name: subject
+      description: The subject of the ticket
+    - name: requester_id
+      description: The numeric ID of the user asking for support through the ticket
+    - name: submitter_id
+      description: The numeric ID of the user submitting the ticket
+    - name: assignee_id
+      description: The numeric ID of the agent to assign the ticket to
+    - name: group_id
+      description: The numeric ID of the group to assign the ticket to
+    - name: body
+      required: true
+      description: Comment body
+    - name: public
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+      description: Does the body public comment
+      defaultValue: "true"
+    - name: type
+      auto: PREDEFINED
+      predefined:
+      - problem
+      - incident
+      - question
+      - task
+      description: Allowed values are problem, incident, question, or task
+    - name: priority
+      auto: PREDEFINED
+      predefined:
+      - urgent
+      - high
+      - normal
+      - low
+      description: Allowed values are urgent, high, normal, or low
+    - name: tags
```

Comment:
@gshriki What do you mean? I just push it and they (their API) automatically divide it). I checked it.